apiserver/admin.go

```diff
@@ -427,7 +427,7 @@ func (a *admin) checkUserPermissions(userTag names.UserTag, controllerOnlyLogin
 		everyoneGroupAccess = everyoneGroupUser.Access
 	}
 
-	controllerAccess := permission.NoAccess
+	var controllerAccess permission.Access
```

Comment:
I disagree with this change - even though permission.NoAccess = "" it's better to be explicit. What happens if we change the value of the const to "none". The code would break.

Reply:
I think the way forward here is to change the logic so that it reads the way people want to read it.